instances/latest/contentTypes/text_vnd.citation-file-format

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/contentTypes/text_vnd.citation-file-format",
+  "@type": "https://openminds.ebrains.eu/core/ContentType",
+  "dataType": null,
+  "description": "CITATION.cff files are plain text files with human- and machine-readable citation information for software (and datasets). Code developers can include them in their repositories to let others know how to correctly cite their software.",
```

Comment:
should we keep this software independent?

Reply:
I would suggest "research product"

Reply:
it is our definition and we can update this to how this format is used across communities